pattern-library/html/boilerplate-blank-RTL.html

```diff
@@ -0,0 +1,54 @@
+<!--
+edX Pattern Library: Boilerplate HTML Template: Header + Footer (RTL layout direction!)
+
+NOTES:
+
+* base template with details on how to handle primary/secondary assets, metadata, and basic view-level UI (on canvas vs. conditional and off-canvas elements)
+* right-to-left (RTL) layout configuration have been added
+* see https://github.com/edx/ux-pattern-library/wiki/Styleguide:-HTML#right-to-left-rtl-support for more information
+-->
+
+<!DOCTYPE html>
+<html class="no-js" lang="en" dir="rtl">
+    <head dir="rtl">
+        <meta charset="utf-8">
+        <!-- always force latest IE rendering engine or request chrome rrame -->
+        <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
```

Comment:
It seems that Google Frame is discontinued so the chrome=1 part is unnecessary too!

http://stackoverflow.com/questions/22059060/is-it-still-valid-to-use-ie-edge-chrome-1

I'm going to remove the whole line as you originally recommended. :-)